virttest/env_process.py

```diff
@@ -1136,7 +1137,8 @@ def preprocess(test, params, env):
         if vm_type == "libvirt":
             if libvirtd_inst is None:
                 libvirtd_inst = utils_libvirtd.Libvirtd()
-            libvirtd_inst.restart()
+            if libvirtd_inst.restart() is None:
```

Comment:
why check for None here, it should be False right, https://github.com/avocado-framework/avocado-vt/blob/4514fad0dba16841f019cc0ae7c0f2a1802b1ffc/virttest/utils_libvirtd.py#L62 , am I missing anything?

More over I feel,  it is better to handle the error cases, in Libvirtd - _wait_for_start(), possibly api taking  additional flag as ignore_status set default to False, and can be set to True if incase needed for negative test sceanrios, so that we do not handle in all the places and this will cover future usage of such api' aswell..

Reply:
Well it should be actually anything but `True`. It returns `False` when it fails to restart and `None` when it won't start on time after the restart.

Reply:
let's just use `if not ..:` everywhere.